client/cli/src/commands/runcmd.rs

```diff
@@ -93,13 +94,19 @@ pub struct RunCmd {
 	#[structopt(long = "unsafe-rpc-external")]
```

Comment:
```suggestion
	#[structopt(long)]
```
You don't need this, it is automagically done for you

Reply:
Same as https://github.com/paritytech/substrate/pull/5729#discussion_r413909683; I didn't touch that in the end.